src/main/kotlin/org/rust/ide/utils/ExprUtils.kt

```diff
@@ -235,10 +232,6 @@ fun RsExpr.skipParenExprUp(): RsExpr {
  *
  * @return a child expression without parentheses.
  */
-fun RsCondition.skipParenExprDown(): RsExpr {
-    var child = this.expr
-    while (child is RsParenExpr) {
-        child = child.expr
-    }
-    return child
-}
+fun RsCondition.skipParenExprDown(): RsExpr = this.expr.skipParenExprDown()
```

Comment:
Whyyyyyyyyyyy did you replaced a loop with a recursion? use `tailrec` modifier at least

Reply:
It isn't a recursive call, because `RsCondition` isn't `RsExpr`.